Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apk add --no-cache curl make && \
 COPY . .
 RUN make dist version=${VERSION}
 
-FROM python:3-alpine3.10
+FROM python:3-alpine3.14
```

Comment:
There is a 3.15 version available..

Reply:
Yep, I realised after I pushed. Will fix.